pkg/k8shandler/reconciler.go

```diff
@@ -35,20 +34,27 @@ func Reconcile(requestCluster *logging.ClusterLogging, requestClient client.Clie
 		clusterLoggingRequest.ForwarderSpec = forwarder.Spec
 	}
 
+	clusterLoggingRequest.Cluster.Status.Conditions.SetCondition(logging.NewCondition("ConditionUnknown",
+		corev1.ConditionUnknown, "Processing", "Reconciliation in progress"))
 	if clusterLoggingRequest.IncludesManagedStorage() {
 		// Reconcile certs
 		if err = clusterLoggingRequest.CreateOrUpdateCertificates(); err != nil {
+			clusterLoggingRequest.Cluster.Status.Conditions.SetCondition(
+				condInvalid("Unable to create or update certificates"))
 			return fmt.Errorf("Unable to create or update certificates for %q: %v", clusterLoggingRequest.Cluster.Name, err)
```

Comment:
This is a hard failure for deploying managed storage but not necessarily log forwarding.  @alanconway thoughts on how best to address?

Reply:
See above

Reply:
ok